src/components/Settings/SettingsPanel/SettingsPanel.tsx

```diff
@@ -90,6 +90,10 @@ export default function SettingsPanel(): JSX.Element {
       id: "mg-MG",
       label: "Malagasy (MG)",
     },
+    {
+      id: "ja-JA",
+      label: "Japanese (JA)",
```

Comment:
Out of scope for this PR, but I wonder if these labels should be the name of the language translated. Here is an example on Twitch. 
<img width="280" alt="Screen Shot 2020-12-21 at 3 40 26 PM" src="https://user-images.githubusercontent.com/4443635/102832337-e7580380-43a2-11eb-8b26-f1df5d860bba.png">